admin_manual/configuration_server/background_jobs_configuration.rst

```diff
@@ -85,6 +85,22 @@ You can verify if the cron job has been added and scheduled by executing::
 
 .. _easyCron: https://www.easycron.com/
 
+Docker
+^^^^^^
+
+On a docker setup, you must attache the running container and invoke the cron.php.
+
+The command looks like this when your nextcloud container was names "my-nextcloud-tld"::
+
+  docker exec $(docker ps  \
+  --filter "Name=my-nextcloud-tld" \
+  --format "{{.Names}}") \
+  su -c "/usr/local/bin/php cron.php" -s /bin/sh www-data
+
+
+Make sure that the container name is unique. Otherwise the filter in the subshell will 
+operate as a wildcard filter and the command will invoke in all matched containers.
```

Comment:
I don't understand?
Yeah, you should use the container name, and it must be unique. In case when you use docker swarm mode, the container name is fluent.

E.g.

```
docker service ls
ybz3xpcosdst        nextcloud            replicated          1/1                 nextcloud:15.0.0                            
```

And when I filter to the container name

```
$ docker ps  --filter "Name=nextcloud" --format "{{.Names}}"
nextcloud.1.ykvb8gcv1pib5uizkdg0zwkuu
```

So this merge request is a generic approach and it depends on your docker setup.

Reply:
Especially in swarm mode you can scale instances and balance the load between them. The command would then return all of them.